package-overrides/npm/parse@1.6.5.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {},
+  "devDependencies": {},
+  "main": "parse-latest.js",
+  "directories": {
+    "lib": "dist"
+  },
```

Comment:
We try to avoid setting this property as it is a breaking change for existing users.